IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,144 @@
+//1. Create a struct called `Flavor`. Add the following properties: Name, Rating Think about what type they should be.
+
+struct Flavor {
+    let name: String
+    let rating: Int
+}
+
+//2. Create an enum called `Size`.
+// Give it a case for small, medium, and large.
+// Make the enum have `Double` raw values. The raw value should represent the size's price. For example, small might be 3.99.
+
+enum Size: Double {
+    case small
```

Comment:
Remember that the instruction said to give these 'Double Raw Values'. For example, small might be 3.99